### DISCUSSION TITLE
Unable to verify the first certificate

### DISCUSSION THREAD

**Discussion by stackerito** [vercel/next.js#29043] [Help]
I created  a self-signed SSL certificate that works well when accessing the website from the browsers.
But now I try to connect to the backend API from inside the Next.js code and I get `Unable to verify the first certificate` error. 

I am using Docker so I try to call my API like so: `fetch('https://backend:443/api/list')`

How can I fix that?

**Comment by chuckcarpenter** [comment]
I used this in the package.json file for Node to read the certs,
```
"dev": "NODE_EXTRA_CA_CERTS=\"$(mkcert -CAROOT)/rootCA.pem\" next dev",
```
